phangsPipeline/__init__.py

```diff
@@ -12,6 +12,8 @@
 from .phangsLogger import setup_logger
 from .handlerKeys import KeyHandler
 from .handlerSingleDish import SingleDishHandler
+# TODO: REMOVE THIS COMMENT AFTER COMPLETION
```

Comment:
Flagging this for future

Reply:
fixed!

There's now an empty `AlmaDownloadHandler` defined when any of the other dependencies are missing.